js/interactive-guides/table-of-contents.js

```diff
@@ -134,14 +135,23 @@ var tableofcontents = (function() {
      * Decide if the guide time duration label needs to be shown.
      */
     var __handleFirstStepContent = function(step) {
+      var isFirstStep;
+      if(step.parent){
+        isFirstStep = __getStepIndex(step.parent.name) === 0;
+      }
+      else{
+        isFirstStep = __getStepIndex(step.name) === 0;
+      }
       // Only show the duration on the first step
-      if(__getStepIndex(step.name) != 0) {
+      if(isFirstStep) {
+        $(ID.toc_guide_title).hide();
+        $(ID.blueprintTitle).show();
+        $(ID.first_step_header).show();       
+        
+      } else {
         $(ID.toc_guide_title).show();
+        $(ID.blueprintTitle).hide();
```

Comment:
Sorry, why do we need to hide the blueprint_title?

Reply:
Each step's title is not using this div with ID.blueprintTitle on it, it creates its own with a class "title". This one is the first step's title with an id on it and it reads "Building fault tolerant microservices with the @CircuitBreaker annotation". We want to hide/show this based on if they are on the first step or not.

Reply:
Got it as I forgot that it has been changed to use .title since I was still viewing the current circuit-breaker guide without your changes. Thanks.